Flutter/cartesian-charts/axis-customization.md

```diff
@@ -586,6 +586,81 @@ The [`visibleMaximum`](https://pub.dev/documentation/syncfusion_flutter_charts/l
 {% endhighlight %}
 {% endtabs %}
 
+## onRendererCreated
+
+Triggers when the axis renderer is created. This callback can be used to obtain the [`AxisController`] instance, which is used to access the the public properties in the axis. Here, you can customize the following properties.
```

Comment:
This callback is triggered when the axis renderer is created and attached to its parent. It can be used to obtain the axis-wise [`AxisController`] which has the current visible range details and provides an option to change the visible range programmatically.